docker/hamer-dev.Dockerfile

```diff
@@ -11,6 +12,9 @@ RUN apt-get install -y --no-install-recommends \
     git \
     && rm -rf /var/lib/apt/lists/*
 
+RUN apt-get update && apt-get install ffmpeg libsm6 libxext6  -y
+RUN apt-get install libglfw3-dev libgles2-mesa-dev
```

Comment:
This line fails with:

```
 ERROR [hamer-dev hamer  5/14] RUN apt-get install libglfw3-dev libgles2-mesa-dev  
 
 ...
 
 2.592 0 upgraded, 33 newly installed, 0 to remove and 2 not upgraded.
2.592 Need to get 14.2 MB of archives.
2.592 After this operation, 79.1 MB of additional disk space will be used.
2.592 Do you want to continue? [Y/n] Abort.
```

This is most likely because the line lacks `-y` to confirm the installations :)

Reply:
Added -y

Reply:
Yeah after that everything did work on my local copy.
It is safe to say that @geopavlakos can apply this PR, and then move to #73